modules/oadp-creating-backup-cr.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+[id="oadp-creating-backup-cr_{context}"]
+= Creating a Backup CR
+
+You back up Kubernetes images, internal images, and persistent volumes (PVs) by creating a `Backup` custom resource (CR).
+
+.Prerequisites
+
+* You must install the OpenShift API for Data Protection (OADP) Operator.
+* The `DataProtectionApplication` CR must be in a `Ready` state.
```

Comment:
Do we need to mention that this CR is associated with OADP? Or will users know what this CR is?

Reply:
Yes. The installation section is in a separate PR. It includes creating the DPA CR.